web-common/src/features/dashboards/leaderboard/DimensionLeaderboardEntrySet.svelte

```diff
@@ -59,31 +67,69 @@ see more button
       ? (filterExcludeMode && active) || (!filterExcludeMode && !active)
       : false;
 
-    return { ...v, active, excluded, comparisonValue };
+    // FIXME: `showComparisonForThisValue` should not be the responsibility
+    // of this component; the handling of the on:mouseenter/on:mouseleave
+    // events should be done in each individual DimensionLeaderboardEntry
+    const showComparisonForThisValue = comparisonLabelToReveal === v.label;
```

Comment:
This whole block is rerun for every `comparisonLabelToReveal` change, i.e. hovering over leaderboard values. Perhaps we should have both versions of formatted string and toggle while displaying. I see there is already a FIXME. Any idea on when we are planning to address it?

Reply:
I don't know, I was going to mention to @nishantmonu51 that these components have accumulated a great deal of tech debt and badly need some refactoring (I left several other FIXMEs in this PR in addition to this one). It is my recommendation to start on that right away, but it's up to Nishant to make the call.

Reply:
@AdityaHegde , discused with Nishant this morning-- the current plan is to tackle these FIXMES after working on the % of total for the dimension table